src/amo/components/VPNPromoBanner/styles.scss

```diff
@@ -0,0 +1,141 @@
+@import '~amo/css/styles';
+
+$cta-text-color: #1d1133;
+$cta-alternate-color: #f9f7fd;
+
+.VPNPromoBanner {
+  background-color: #c5c8fb;
+  font-family: sans-serif;
+  font-size: 16px;
+  line-height: $line-height-compressed;
+  max-height: 500px;
+  overflow: hidden;
+  position: relative;
+  transition: max-height 0.3s ease-out;
+}
+
+@media (prefers-reduced-motion) {
```

Comment:
It's a good idea to add that but why not adding it to all transitions?

Reply:
As above, I pretty much just implemented what already existed. I am guessing, for the other transitions in this promo, it wasn't considered important. Fir example, I don't think we'd need to prevent the background colour of the button from transitioning, as it doesn't involve any motion. For the dismisser button, we can implement it (I'll do that in an update), but I guess maybe the amount of movement and the time it takes for the transition are both so slight it might not have been considered important.